pkg/cmd/admin/router/router.go

```diff
@@ -293,6 +299,15 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 					{Type: dapi.DeploymentTriggerOnConfigChange},
 				},
 				Template: dapi.DeploymentTemplate{
+					Strategy: dapi.DeploymentStrategy{
```

Comment:
Isn't rolling the default? Why are we copying in a bunch of default parameter values? Just to get the down-first, percent-based behavior?